exampleAuthentication/1.authentication.js

```diff
@@ -65,7 +65,7 @@ console.log("entity1Puk pubk:", '04'+entity1Puk.getPublicKeyString().substr(2))
   console.log('\tIs the signedJWT verified?', verifyAT)
 
 
-const alastriaSession = tokensFactory.tokens.createAlastriaSession(context, configData.didSubject1, "04"+entity1Puk.getPublicKeyString().substr(2), signedAT, tokenExpTime, tokenActivationDate, jsonTokenId)
+const alastriaSession = tokensFactory.tokens.createAlastriaSession(context, configData.didSubject1, "04"+subject1Pubk.getPublicKeyString().substr(2), signedAT, tokenExpTime, tokenActivationDate, jsonTokenId)
```

Comment:
I think this should be without "04" and with "0x"
```suggestion
const alastriaSession = tokensFactory.tokens.createAlastriaSession(context, configData.didSubject1, subject1Pubk.getPublicKeyString(), signedAT, tokenExpTime, tokenActivationDate, jsonTokenId)
```

Reply:
Changed